api/src/main/java/org/openmrs/module/appointments/AppointmentsActivator.java

```diff
@@ -25,16 +25,10 @@
 	
 	private Log log = LogFactory.getLog(this.getClass());
 	
-	/**
-	 * @see org.openmrs.module.Activator#startup()
```

Comment:
We had to remove these lines because there were errors from `mvn-javadoc-plugin` as `Cannot find definition`.